grails-app/controllers/grails/plugins/mail/oauth/MailOAuthController.groovy

```diff
@@ -12,86 +12,107 @@ import com.github.scribejava.core.model.OAuth2AccessTokenErrorResponse
 class MailOAuthController {
 
     String uri = Holders.config.grails.mail.oAuth.redirect.uri
-
     def mailOAuthService
 
     def index() {
+        log.debug("[GRAPH_EMAIL] [INDEX] - Accessed index endpoint")
     }
 
     def generate() {
         if (!grailsApplication.config.grails.mail.oAuth.enabled) {
+            log.warn("[GRAPH_EMAIL] [GENERATE] - OAuth configuration is disabled")
             flash.warn = "Please enable mail OAuth configuration"
             redirect(uri: uri)
             return
         }
-        log.debug("Requested for new AuthToken")
+
+        log.info("[GRAPH_EMAIL] [GENERATE] - Requested new AuthToken | Redirecting to Auth URL")
```

Comment:
@RxL-Farhan-Ali Keep it at debug level not info. If any application require then it can open debug level.